docs/reference/webserver.md

```diff
@@ -0,0 +1,32 @@
+# Using the Doorstop web front-end
+
+## Conventional bottle webserver
+
+Doorstop can be run as a standalone web server by running
+`python3 doorstop/server/main.py`.  It will use the current working
```

Comment:
This is actually exposed as a command-line client, `doorstop-server`. I think that should be the recommended approach here.